go/heif/heif.go

```diff
@@ -273,13 +273,9 @@ func convertHeifError(cerror C.struct_heif_error) error {
 		Subcode: ErrorSubcode(cerror.subcode),
 		Message: C.GoString(cerror.message),
 	}
-	if err.Code == ErrorUnsupportedFiletype {
-		fmt.Println("XXX")
-	}
-	return err
```

Comment:
The `return err` should remain (see Travis errors).

Reply:
Oopsie, fixed.